src/icons/flags/AD.tsx

```diff
@@ -3,9 +3,9 @@ import { forwardRef } from 'react';
 import { FlagIconBase } from 'src/icons/flag-icon/_FlagIconBase';
 
 type Props = {
-  borderRadius?: number;
   height: number;
   width: number;
+  borderRadius?: number;
```

Comment:
How come this prop is not alphabetically sorted? is this the new rule where require prop has higher priority? :thinking: